addons/hr_expense/tests/test_expenses.py

```diff
@@ -608,6 +609,7 @@ def test_print_expense_check(self):
         sheet.approve_expense_sheets()
         sheet.action_sheet_move_create()
         action_data = sheet.action_register_payment()
+        action_data['context']['default_group_payment'] = False
```

Comment:
Don't change it in the context, but rather in the Form view, like a user would do (since we are using the Form view emulator here)
But does this mean that the default behavior is not tested?